tools/_tools.font-size.scss

```diff
@@ -5,6 +5,14 @@
 // Generate a font-size and baseline-compatible line-height.
 @mixin inuit-font-size($font-size, $line-height: auto, $important: false) {
 
+  @if (type-of($font-size) == number) {
+    @if (unit($font-size) != "px") {
+      @error "`#{$font-size}` needs to be a pixel value.";
+    }
+  } @else {
+    @error "`#{$font-size}` needs to be a number.";
+  }
+
```

Comment:
@csshugs I blindly assumed that `sass-true` (should’ve guessed differently from the name already) supports _expect failure_ as well. It doesn’t, so we can’t test this. Please ignore all my comments on this then.